lib/screens_switcher.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:mentorship/core/theming/colors.dart';
+import 'package:mentorship/features/home/logic/home_cubit.dart';
+import 'package:mentorship/features/home/ui/screens/home_screen.dart';
+import 'package:persistent_bottom_nav_bar/persistent_bottom_nav_bar.dart';
+
+class ScreensSwitcher extends StatelessWidget {
```

Comment:
ScreensSwitcher isn't the proper name I guess.

Reply:
I thought about it's functionality (to have the bottom nav bar and to switch between screens) But ig MainScreen is better?